test/parallel/test-file-write-stream3.js

```diff
@@ -74,13 +72,12 @@ function run_test_1() {
 
 
 function run_test_2() {
-  var file, buffer, options;
 
-  buffer = Buffer.from('123456');
+  const buffer = Buffer.from('123456');
 
-  options = { start: 10,
+  const options = { start: 10,
               flags: 'r+' };
```

Comment:
Can you line this up so that start and flags line up like they did before (ie need to indent flags to align with new position of start)